StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Controllers/PayWithLinkWebController.swift

```diff
@@ -72,23 +72,27 @@ final class PayWithLinkWebController: NSObject, ASWebAuthenticationPresentationC
         let configuration: PaymentElementConfiguration
         let callToAction: ConfirmButton.CallToActionType
         var lastAddedPaymentDetails: ConsumerPaymentDetails?
+        let alwaysUseEphemeralSession: Bool
 
         /// Creates a new Context object.
         /// - Parameters:
         ///   - intent: Intent.
         ///   - elementsSession: STPElementsSession.
         ///   - configuration: PaymentElementConfiguration configuration.
         ///   - callToAction: A custom CTA to display on the confirm button. If `nil`, will display `intent`'s default CTA.
+        ///   - alwaysUseEphemeralSession: If `true`, always use an ephemeral session.
```

Comment:
what is the significance of 'always' here?  Seems like it could just be called useEphemeralSession?

Reply:
https://github.com/stripe/stripe-ios/pull/4435/files#diff-3586b686dbee34f78db7cb9f98263219ac168f2ead0b422686cab88b80f767f8R144